lib/aareadme.txt

```diff
@@ -106,6 +106,11 @@ groovy-3.0.3
 ----------------------
 Advised scripting language for JSR223 Test Elements
 
+gson-2.8.6
+----------------------
+https://github.com/google/gson
```

Comment:
Wasn't it possible to use Jackson which is already a dependency of JMeter ?
https://www.baeldung.com/jackson-vs-gson

Reply:
Fixed: f9a10107a
Thanks!